<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DarkCode</class>
 <widget class="QMainWindow" name="DarkCode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DarkCode</string>
  </property>
  <property name="locale">
   <locale language="Chinese" country="China"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget_headline" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#widget_headline{
background:#0A0A0A;
border-top-left-radius:8px;
border-top-right-radius:8px;
border:1px solid;
border-top-color:rgb(102.2, 177.4, 255);
border-right-color:rgb(133.4, 206.2, 97.4);
border-left-color:rgb(235, 180.6, 99);
border-bottom-color:#E6DB74;
}</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_softname" native="true">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_softname">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">#label_softname{
color:#409EFF;
font-size:16px;
font-weight:700;
letter-spacing:1.5px;
}</string>
            </property>
            <property name="text">
             <string>DarkCode</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_documentname" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_documentname">
            <property name="styleSheet">
             <string notr="true">#label_documentname{
color:#909399;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_threebutton" native="true">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_min">
            <property name="minimumSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">#pushButton_min{
border-radius:8px;
background:#F56C6C;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_max">
            <property name="minimumSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">#pushButton_max{
border-radius:8px;
background:#E6A23C;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_quit">
            <property name="minimumSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">#pushButton_quit{
border-radius:8px;
background:#67C23A;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_mainwindow" native="true">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">#widget_mainwindow{
background:#272822;
border-bottom-left-radius:8px;
border-bottom-right-radius:8px;
border:1px solid;
border-top:none;
border-right-color:rgb(133.4, 206.2, 97.4);
border-bottom-color:rgb(247, 137.4, 137.4);
border-left-color:rgb(235, 180.6, 99);
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_work" native="true">
         <property name="styleSheet">
          <string notr="true">background:transparent;</string>
         </property>
         <widget class="QToolBox" name="toolBox_code_files">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>161</width>
            <height>551</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>-1</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolBox{font-size:14px;font-weight:700;}

QToolBox::tab{background-color:transparent;color:white;}

QToolBox::tab:selected{background-color:#67C23A;font-weight:700;}

QToolBox::tab:hover:!selected{background-color:rgb(209.4, 236.7, 195.9);}</string>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <property name="tabSpacing">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page_setting">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>161</width>
             <height>499</height>
            </rect>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <attribute name="label">
            <string>设置</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QListWidget" name="listWidget_setting">
              <property name="styleSheet">
               <string notr="true">#listWidget_setting{
color:white;
font-size:14px;
font-weight:700;
}
#listWidget_setting::item{
margin-top:10px;
}</string>
              </property>
              <item>
               <property name="text">
                <string>Python环境配置</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>快捷键配置</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_program_file">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>161</width>
             <height>499</height>
            </rect>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <attribute name="label">
            <string>项目文件夹</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTreeWidget" name="treeWidget_program_file">
              <property name="styleSheet">
               <string notr="true">#treeWidget_program_file{
color:white;
font-size:14px;
font-weight:700;
}
QHeaderView{
font-size:14px;
font-weight:700;
color:#67C23A;
}</string>
              </property>
              <column>
               <property name="text">
                <string notr="true">1</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QTabWidget" name="tabWidget_code_pages">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>0</y>
            <width>841</width>
            <height>551</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QTabWidget::pane{
background:transparent;
border:1px solid;
border-right-color:rgb(133.4,206.2,97.4);
}

QTabBar::tab{background-color: #0B0E11;color:white;}

QTabBar::tab:selected{background-color:#409EFF;}

QTabBar::tab:hover:!selected{background-color:rgb(216.8,235.6,255);}

QTabBar QToolButton{border:none;color:rgb(255,206,6);background-color:#0b0e11;}

QTabBar QToolButton:hover{background-color: #161a1e;}</string>
          </property>
          <property name="currentIndex">
           <number>-1</number>
          </property>
          <property name="movable">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_status" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">#widget_status{
background:#4C4D4F;
border-bottom-left-radius:10px;
border-bottom-right-radius:10px;
border:1px solid;
border-right-color:rgb(133.4, 206.2, 97.4);
border-bottom-color:rgb(247, 137.4, 137.4);
border-left-color:rgb(235, 180.6, 99);
border-top-color:#E6DB74;
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
